src/v0/util/index.js

```diff
@@ -47,7 +47,13 @@ const flattenMap = (collection) => _.flatMap(collection, (x) => x);
 // GENERIC UTLITY
 // ========================================================================
 
-const getEventTime = (message) => new Date(message.timestamp).toISOString();
+const getEventTime = (message) => {
+  try {
+    return new Date(message.timestamp).toISOString();
+  } catch (err) {
+    return new Date(message.originalTimestamp).toISOString();
```

Comment:
@sanpj2292  last time we discussed that we shouldn't use original timestamp post server, why are we using here?

Reply:
I think that was the fix & it was decided by the team. I just raised a PR for hotfix
@Sanjay-Veernala or @krishna2020  can you answer @koladilip 's query ?

Reply:
It's being used only as fallback, otherwise nothing goes.

Reply:
In the cases where user sends `timestamp` as part of payload, server passes it as-is. So in case of empty timestamp being sent in the payload, this scenario can occur.
Instead rudder-server can validate & put the current timestamp otherwise